src/pages/layer-2.js

```diff
@@ -0,0 +1,618 @@
+          done a great job addressing decentralization, and security, with plans
+          to address scalability with sharding in the future. While there are
+          roadmap plans to improve the scalability of Ethereum, there is still a
+          need for further scaling to round out the trilemma for Ethereum. This
+          is where Layer 2 technology is being developed to step in and bring
+          further scaling to Ethereum. As time has progressed, Ethereum as a
+          layer 1 is a settlement layer for layer 2s, and not the layer for end
+          users.
+        </p>
+      </Content>
+
+      <Content>
+        <h2>Why do we need layer 2?</h2>
+        <p>
+          The three desirable properties of a blockchain are that it is
+          decentralized, secure, and scalable. The scalability trilemma states
+          that a simple blockchain architecture can only achieve two out of
+          three. Want a secure and scalable blockchain? You need to sacrifice
+          decentralization.
+        </p>
+        <p>
+          Ethereum has reached the networks capacity with{" "}
+          <Link to="https://etherscan.io/chart/tx">
+            1+ million transactions per day
+          </Link>
+          , with high demand for each of these transactions. Due to the success
+          of Ethereum and the demand to use it, gas prices have risen
+          substantially, increasing the{" "}
+          <Link to="/developers/docs/scaling/">need for scaling solutions</Link>
+          . The main goal of scalability is to increase transaction speed
+          (faster finality), and transaction throughput (high transactions per
+          second), without sacrificing decentralization or security (more on the
+          Ethereum vision).
+        </p>
+        <p>
+          The Ethereum community has taken a strong stance that it would not
+          throw out decentralization or security in order to scale. Until
+          sharding, Ethereum Mainnet (layer 1) will only be able to process
+          roughly 15 transactions per second. When demand to use Ethereum is
+          high this causes network congestion, increasing transaction fees, and
+          pricing out those who cannot afford it from using Ethereum until the
+          fees reduce. That is where layer 2 comes in.
+        </p>
+      </Content>
+
+      <Content>
+        <InfoGrid>
+          {layer2Cards.map(({ image, title, preview, body, alt }, idx) => (
+            <ExpandableCard
+              contentPreview={preview}
+              title={title}
+              svg={image}
+              alt={alt}
+              key={idx}
+            >
+              {body.map((item, idx) => (
+                <p key={idx}>{item}</p>
+              ))}
+            </ExpandableCard>
+          ))}
+        </InfoGrid>
+      </Content>
+
+      <Content>
+        <h2>Layer 2</h2>
+        <p>
+          Layer 2 refers to building other Ethereum compatible networks on top
```

Comment:
> Layer 2 refers to building other Ethereum compatible networks on top of layer 1, and requires no changes to the layer 1 protocol. 

We need a stronger definition than this. Perhaps something along the lines of....

> Layer 2 is a collective term for Ethereum scaling solutions that handle transactions off Ethereum layer 1 while taking advantage of the robust decentralized security model of Ethereum layer 1.

We may also want to be clear about what specifically a L2 is - a blockchain. Something like:

> A layer 2 is separate blockchain that is connected to Ethereum. It regularly communicates with Ethereum in order to ensure it has similar security and decentralization guarantees.